packages/grid/x-data-grid/src/components/panel/filterPanel/GridFilterInputValueProps.ts

```diff
@@ -1,10 +1,11 @@
 import * as React from 'react';
+import { TextFieldProps } from '@mui/material/TextField';
 import { GridFilterItem } from '../../../models/gridFilterItem';
 
-export interface GridFilterInputValueProps {
+export type GridFilterInputValueProps<T> = {
```

Comment:
Can we type it better?

```ts
import type { GridApiCommon } from '../../../models/api/gridApiCommon';
import type { GridApiCommunity } from '../../../models/api/gridApiCommunity';

export type GridFilterInputValueProps<Api extends GridApiCommon = GridApiCommunity> = {
  // ...
  apiRef: React.MutableRefObject<Api>;
```

Maybe something like this?
The `Api` is optional, so no need for `GridFilterInputValueProps<any>` in other places
What do you think?

Reply:
Correct